### PR TITLE
Special code is not needed for exhibitors, this was removed

### PR DESCRIPTION
**Pull request by Jeppes**
Removed unneeded synching code for exhibitors. All caching is handled the same way now

### CODE REVIEW

File: app/src/main/java/se/ths/kth/Aramda/data/DataManager.java [hunk 1]
```diff
@@ -91,39 +87,22 @@ public void toggleExhibitorFavorited(int id) {
     }
 
     @Override
-    public void insertFromJSONSync(String json, @API.ApiCalls String endpoint, final Class<? extends RealmObject> clazz) throws JSONException {
+    public void insertFromJSONSync(String json, final Class<? extends RealmObject> clazz) throws JSONException {
         // This method is run on a background thread, it needs its own realm instance
         Realm realmAsync = Realm.getDefaultInstance();
+
+        // Remove all objects of the current type
+        deleteAll(realmAsync, clazz);
```
Comment (apals) [inline app/src/main/java/se/ths/kth/Aramda/data/DataManager.java]: do we need to delete everything. we do `createAllOrUpdate`

Comment (Jeppes) [inline app/src/main/java/se/ths/kth/Aramda/data/DataManager.java]: delete everything. When the new companies are inserted we don't want any old companies hanging around (that are either not attending this year or have been assigned a different id)

Comment (Jeppes) [inline app/src/main/java/se/ths/kth/Aramda/data/DataManager.java]: Otherwise we'll need a smarter syncing system in the backend

Comment (Jeppes) [inline app/src/main/java/se/ths/kth/Aramda/data/DataManager.java]: same goes for events and partners of course

**Pull request merged**